cypress/utils/utils.ts

```diff
@@ -504,6 +538,44 @@ export function verifySortDesc(listToVerify: Array<any>, unsortedList: Array<any
     });
 }
 
+export function verifyDateSortAsc(listToVerify: Array<string>, unsortedList: Array<string>): void {
+    cy.wrap(listToVerify).then((capturedList) => {
+        let sortedList = [...unsortedList]
```

Comment:
This variable is not re-defined, and can be declared as a `const`.

On the other hand, `unsortedList` already is an array so I think there is no need to spread it.

Replace

`[...unsortedList]`

with `unsortedList` and it should have the same effect (please try it before commiting)

Same comment for the `verifyDateSortDesc` method

Thanks!